lib/cusparse/interfaces.jl

```diff
@@ -17,7 +17,7 @@ LinearAlgebra.mul!(C::CuVector{T},adjA::Adjoint{<:Any, <:HermOrSym{T,<:CuSparseM
 
 function mm_wrapper(transa::SparseChar, transb::SparseChar, alpha::Number,
                     A::CuSparseMatrix{T}, B::CuMatrix{T}, beta::Number, C::CuMatrix{T}) where {T}
-    if version() < v"10.3.1" && A isa CuSparseMatrixCSR
+    if version() < v"10.3.1" && A isa CuSparseMatrixCSC
         # generic mm! doesn't work on CUDA 10.1 with CSC matrices
```

Comment:
The comment was wrong or not?

Reply:
I don't remember. Let CI be the judge!

Reply:
... and CI failed for 10.1, so I guess the comment was wrong.

Reply:
Ok, I fixed the typo.